osf/utils/workflows.py

```diff
@@ -62,3 +62,7 @@ class DefaultTriggers(ChoiceEnum):
         'after': ['save_action', 'save_changes', 'notify_edit_comment'],
     },
 ]
+
+@unique
+class RequestTypes(ChoiceEnum):
```

Comment:
Just a question - does this belong here, or perhaps in `request.py` as it seems to be very `AbstractRequest` specific and maybe not needed by other models?

Reply:
Could've gone either way with this. I opted to put it here for three main reasons:
* It seemed to fit more with `utils` than with `models`, as it's not stored in the database
* `AbstractRequest` objects are machineable, and machines have workflow-related metadata that governs their behavior. This seemed to qualify as "workflow-related metadata," the rest of which resides in this file.
* This may need to be checked in certain places where `AbstractRequests` might not need to be imported. At present, it's not, but there could be cases where doing an import from the model file would cause a circular import

Given that, if you think I should put it with the model, I will.

Reply:
Naw, makes sense!